cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,33 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        db_conn = None
+        max_attempts = 5
+        attempts = 0
+
+        while not db_conn and attempts < max_attempts:
+            try:
+                connections["default"].ensure_connection()
+                db_conn = connections["default"]
+            except OperationalError:
+                attempts += 1
+                self.stdout.write(
+                    f"Database unavailable, waiting 1 second... "
+                    f"(Attempt {attempts}/{max_attempts})"
+                )
+                time.sleep(1)
```

Comment:
In the situation where the database is not available, you're simply waiting for 1 second and then trying again. However, this may not be the most efficient approach, especially if the database is likely to be unavailable for a longer period of time. Consider implementing a backoff strategy, which would increase the wait time between each attempt. This way, your application won't waste resources by trying to connect to the database too frequently when it's not available.